### PR TITLE
[fix bug 1130194] Add FTU tracking to Hello product page

### PR DESCRIPTION
**Pull request by ckprice**
https://bugzilla.mozilla.org/show_bug.cgi?id=1130194

@jpetto r?

### CODE REVIEW

File: media/js/firefox/hello/index.js [hunk 1]
```diff
@@ -127,6 +136,13 @@
                     $('.try-hello').on('click', function(e) {
                         e.preventDefault();
 
+                        // (bug 1115227) pass source to FTU
+                        if (tourSource === 'none') {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_productPage');
+                        } else {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_' + tourSource);
```
Comment (alexgibson) [inline media/js/firefox/hello/index.js]: @ckprice - seeing as these are page id's registered with Telemetry, I think we should try and retain tighter control over what is set, as opposed to setting arbitrary values passed via URL parameter. Thoughts?

Comment (ckprice) [inline media/js/firefox/hello/index.js]: Registering the pageID for the `none` case was superfluous as there is nothing to track. I will remove it.

We're now only registering a page ID if `utm_content` is populated and the Hello panel is opened. I don't think it make sense for people sharing the URL to have to enter some kind of special prefix to attribute their channel work in the FTU.

**Comment by ckprice** [comment]
@alexgibson tightened up the conditions a bit. Thoughts?

**Comment by alexgibson** [comment]
@ckprice - I wasn't aware tagging an external campaign was a requirement here. I thought we we're simply letting the FTE know the referral was from the product page.

This change still allows people to create new Telemetry buckets simply by passing/sharing a URL with a given query param. Ideally I think we should be registering these page ID's ourselves in the code, instead of letting external folks create them.

I don't quite know what the potential risk might be that we should be aware of (if any), so I would like to seek Matt's opinion on this before merging.

**Comment by alexgibson** [comment]
@ckprice - instead of appending any arbitrary value to the page ID, could we use a limited set of ID's we specify in the page, based on the value of the param? I understand this may not give as fine-grained tracking, but I think it's best to have a middle-ground somewhere.

**Comment by ckprice** [comment]
@alexgibson thoughts?

**Comment by alexgibson** [comment]
LGTM, thanks @ckprice! r+

**Pull request merged**